packages/react-native/src/primitives/PhoneNumberField/types.ts

```diff
@@ -42,6 +42,7 @@ export interface PhoneNumberFieldProps
 }
 
 export interface PhoneNumberFieldStyles extends Partial<TextFieldStyles> {
+  fieldStyle: ViewStyle;
```

Comment:
Think this can just be `field`?

Reply:
Yes. Also, these styles aren't even used in `PhoneNumberField`, so I removed them